src/main/kotlin/lotto/domain/WinningStatistics.kt

```diff
@@ -1,6 +1,11 @@
 package lotto.domain
 
-class WinningStatistics(private val statistics: Map<WinningCategory, Int>) {
+class WinningStatistics(tickets: LottoTickets, private val winningLotto: WinningLotto) {
```

Comment:
WinningStatistics의 관심사에 winningLotto를 있을 필요가 있을까요?
당첨번호로 로또 결과 만드는것과 로또 결과 분석을 분리해볼수 있을거같아요!

```suggestion
class WinningStatistics(private val winningCategories: List<WinningCategory >) {
```

Reply:
현재 역할과 책임이 분리가 제대로 되어있지 않은것같습니다 🥲
리팩토링과정에서 당첨번호로 로또 결과 만드는 역할은 WinningLotto로, 로또 결과 분석은 WinningStatistics로 분리했습니다! 🙂